digitaltwins/samples/service/javascript/update_component.js

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+const IoTHubTokenCredentials = require('azure-iot-digitaltwins-service').IoTHubTokenCredentials;
+const DigitalTwinServiceClient = require('azure-iot-digitaltwins-service').DigitalTwinServiceClient;
+const { inspect } = require('util');
+
+const patch = [{
+  interfaces: {
+    'environmentalSensor': { // for the environmental sensor, try "environmentalSensor"
+      properties: {
+        'brightness': { // for the environmental sensor, try "brightness"
+          desired: {
+            value: 42 // for the environmental sensor, try 42 (note that this is a number, not a string, so don't include quotes).
+          }
+        }
+      }
+    }
+  }
+}];
```

Comment:
So I'm presuming that the CL layer transforms this into the JSON patch description?

Reply:
Clearly I presume wrong.

---
In reply to: [450407004](https://github.com/Azure/azure-iot-sdk-node/pull/837#discussion_r450407004) [](ancestors = 450407004)

Reply:
No, it does not

---
In reply to: [450411859](https://github.com/Azure/azure-iot-sdk-node/pull/837#discussion_r450411859) [](ancestors = 450411859,450407004)